terracumber_config/tf_files/Uyuni-Master-podman.tf

```diff
@@ -249,7 +245,7 @@ resource "null_resource" "configure_quality_intelligence" {
   }
 
   provisioner "remote-exec" {
-    inline = [ "echo export QUALITY_INTELLIGENCE=true >> ~/.bashrc",
+    inline = [ "echo export QUALITY_INTELLIGENCE=false >> ~/.bashrc",
```

Comment:
If that's just a pipeline to test your changes fine, but if you plan to merge it, please keep this value to true.

Reply:
Yes, I plan to do that.